sdk/storage/Azure.Storage.Blobs/src/Models/BlobProperties.cs

```diff
@@ -0,0 +1,280 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Azure.Storage.Blobs.Models;
+
+namespace Azure.Storage.Blobs.Models
+{
+    /// <summary>
+    /// Propeties of a Blob
+    /// </summary>
+    public partial class BlobProperties
+    {
+        /// <summary>
+        /// Internal Blob Properties
+        /// </summary>
+        internal BlobPropertiesInternal _properties;
+
+        /// <summary>
+        /// Returns the date and time the blob was last modified. Any operation that modifies the blob, including an update of the blob's metadata or properties, changes the last-modified time of the blob.
+        /// </summary>
+        public System.DateTimeOffset LastModified { get; internal set; }
```

Comment:
I think we might be able to do
```
LastModified => _properties.LastModified;
CreatedOn => _properties.CreatedOn;
etc.
```
And then just take a BlobPropertiesInternal in the constructor.

Reply:
I did try to do this, but in ToBlobProperies method in BlobExtension I ran into an error about these values being readonly, when we are attempting to set them.

Reply:
Oh gotcha.  Sorry I gave conflicting advise on this one, lets leave it as-is.